tests/post/steps/test_logging.py

```diff
@@ -56,6 +72,49 @@ def _check_loki_ready():
     )
 
 
+@given(parsers.parse("we have set up a '{pod_name}' pod"),
+       target_fixture='pod_creation_ts')
+def set_up_logger_pod(host, nodename, k8s_client, utils_image, pod_name):
+    manifest_template = LOGGER_POD_TEMPLATE.read_text(encoding="utf-8")
+    manifest = string.Template(manifest_template).substitute(
+        name=pod_name,
+        image=utils_image,
+    )
+    manifest_path = str(MANIFESTS_PATH / "{}.yaml".format(pod_name))
+
+    with host.sudo():
+        write_manifest = utils.write_string(host, manifest_path, manifest)
+        assert write_manifest.rc == 0, (
+            "Failed to write logger Pod manifest '{}': {}"
+        ).format(manifest_path, write_manifest.stderr)
+
+        pod_creation_ts = int(time.time())
+        result = host.run('kubectl apply -f %s', manifest_path)
```

Comment:
Wow, it's not really good here, you put the file iin the `/etc/kubernetes/manifest/` directory and you kubectl apply, you should only do the kubectl apply otherwise I think here you will create 2 pods: 
- one static pod `logger-test-bootstrap` on the bootstrap node (because of a YAML in the `/etc/kubernetes/manifest` directory)
- one pod `logger-test` on a "random" node (because of the kubectl apply)

IMHO you should use the same approach as in `tests/post/steps/test_dns.py` it's would be simpler

Reply:
True, don't know what I had in mind, I may need some sleep. :D